projects/js-packages/connection/components/disconnect-dialog/test/step-disconnect.jsx

```diff
@@ -47,13 +52,37 @@ describe( 'StepDisconnect', () => {
 
 	describe( 'When the dismiss button is clicked', () => {
 		const wrapper = shallow( <StepDisconnect { ...testProps } /> );
-
-		wrapper
-			.find( '.jp-connection__disconnect-dialog__btn-dismiss' )
-			.simulate( 'click', { preventDefault: () => undefined } );
+		const dismissBtn = wrapper.find( '.jp-connection__disconnect-dialog__btn-dismiss' );
 
 		it( 'calls the closeModal callback', () => {
+			dismissBtn.simulate( 'click', { preventDefault: () => undefined } );
+
 			expect( testProps.closeModal.called ).to.be.true;
 		} );
+
+		it( 'calls the trackModalClick callback with stay_connected', () => {
+			dismissBtn.simulate( 'click', { preventDefault: () => undefined } );
+
+			expect( testProps.trackModalClick.calledOnceWith( 'stay_connected' ) ).to.be.true;
+		} );
+	} );
+
+	describe( 'When help links are clicked', () => {
+		const learnAboutLink = 0;
+		const supportLink = 1;
+		const wrapper = shallow( <StepDisconnect { ...testProps } /> );
+		const links = wrapper.find( '.jp-connection__disconnect-dialog__link' );
+
+		it( 'the connection link calls the trackModalClick with learn_about', () => {
+			links.at( learnAboutLink ).simulate( 'click', { preventDefault: () => undefined } );
```

Comment:
Matching links by index seems like it could be fragile if the link order or content changes in the future.
I wonder if we'd be able to find the components `<jpConnectionInfoLink>` or `<jpSupportLink>`. I haven't tested that out, but that may be one way to specifically identify the links to test.

Reply:
I agree they're a little brittle as written - but the short answer to your question is no.

`wrapper.debug()` shows the rendered html and in a quick test it wasn't possible to find these components.

Reply:
I've changed to a loop which removes the brittleness,  while ensuring all events are tracked.